debian/golang-go.links

```diff
@@ -1,3 +1,3 @@
+usr/share/go/include /usr/lib/go/include
```

Comment:
Ah well, not quite: this line currently just creates a broken symlink:

```
root@go15-tests:~# ls -l /usr/lib/go/include  
lrwxrwxrwx 1 root root 22 Aug 11 23:43 /usr/lib/go/include -> ../../share/go/include
```

textflag.h gets installed to /usr/lib and things work, but maybe it should be /usr/share and then this line fixed as i suggested...